packages/peregrine/lib/talons/CreateAccount/useCreateAccount.js

```diff
@@ -61,8 +62,8 @@ export const useCreateAccount = props => {
         }
     );
 
-    const [signIn, { error: signInError }] = useMutation(signInMutation, {
-        fetchPolicy: 'no-cache'
+    const [signIn, { error: signInError }] = useCreateSignIn({
+        signInMutation: signInMutation
```

Comment:
Not 100% sure but there seems to be a convention where we call it `query: signInMutation`